cm-api-shim.js

```diff
@@ -51,41 +50,50 @@
     return oldGet(options);
   }
 
-  // Polyfill for navigator.credentials.create.
-  navigator.credentials.create = function(options) {
-    var error_msg = "NotSupportedError: Only 'password' and 'federated'" +
-        'credential types are currently supported.';
+  // Polyfill for Credential Management API's navigator.credentials.create.
+  // Other APIs might define this method already, in which case we need to cache
+  // the original version.
+  if (navigator.credentials.create) {
```

Comment:
I added this logic to be able to dispatch to an already existing version of navigator.credentials.create if applicable. WDYT?

Reply:
I like it. LGTM